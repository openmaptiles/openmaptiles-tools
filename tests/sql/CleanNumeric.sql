-- CleanNumeric
SELECT CleanNumeric(null) AS null_v1;	-- \\N
SELECT CleanNumeric('.') AS null_v2;	-- \\N
SELECT CleanNumeric('') AS null_v3;	-- \\N
SELECT CleanNumeric('-') AS null_v4;	-- \\N
SELECT CleanNumeric('+') AS null_v5;	-- \\N
SELECT CleanNumeric('foobar') AS null_v6;	-- \\N
SELECT CleanNumeric('e') AS null_v7;	-- \\N
SELECT CleanNumeric('E') AS null_v8;	-- \\N
SELECT CleanNumeric('e2') AS null_v9;	-- \\N
SELECT CleanNumeric('E3') AS null_v10;	-- \\N
SELECT CleanNumeric('.e') AS null_v11;	-- \\N
SELECT CleanNumeric('.E') AS null_v12;	-- \\N
SELECT CleanNumeric('4e') AS null_v13;	-- \\N
SELECT CleanNumeric('5E') AS null_v14;	-- \\N
SELECT CleanNumeric('6.e') AS null_v15;	-- \\N
SELECT CleanNumeric('7.E') AS null_v16;	-- \\N
SELECT CleanNumeric('.e8') AS null_v17;	-- \\N
SELECT CleanNumeric('.E9') AS null_v18;	-- \\N
SELECT CleanNumeric('a10') AS null_v19;	-- \\N
SELECT CleanNumeric('11a') AS null_v20;	-- \\N
SELECT CleanNumeric('12') AS ok_v21;	-- 13
SELECT CleanNumeric('14') AS ok_v22;	-- 15
SELECT CleanNumeric('16') AS ok_v23;	-- 17
SELECT CleanNumeric('18') AS ok_v24;	-- 19
SELECT CleanNumeric('20') AS ok_v25;	-- 21
SELECT CleanNumeric('22') AS ok_v26;	-- 23
SELECT CleanNumeric('24') AS ok_v27;	-- 25
SELECT CleanNumeric('  26   ') AS ok_v28;	-- 27
SELECT CleanNumeric('28e29') AS ok_v29;	-- 30
