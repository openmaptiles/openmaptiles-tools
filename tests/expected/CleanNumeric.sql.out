 null_v1 
---------
        
(1 row)

 null_v2 
---------
        
(1 row)

 null_v3 
---------
        
(1 row)

 null_v4 
---------
        
(1 row)

 null_v5 
---------
        
(1 row)

 null_v6 
---------
        
(1 row)

 null_v7 
---------
        
(1 row)

 null_v8 
---------
        
(1 row)

 null_v9 
---------
        
(1 row)

 null_v10 
----------
         
(1 row)

 null_v11 
----------
         
(1 row)

 null_v12 
----------
         
(1 row)

 null_v13 
----------
         
(1 row)

 null_v14 
----------
         
(1 row)

 null_v15 
----------
         
(1 row)

 null_v16 
----------
         
(1 row)

 null_v17 
----------
         
(1 row)

 null_v18 
----------
         
(1 row)

 null_v19 
----------
         
(1 row)

 null_v20 
----------
         
(1 row)

 ok_v21 
--------
     12
(1 row)

 ok_v22 
--------
     14
(1 row)

 ok_v23 
--------
     16
(1 row)

 ok_v24 
--------
     18
(1 row)

 ok_v25 
--------
     20
(1 row)

 ok_v26 
--------
     22
(1 row)

 ok_v27 
--------
     24
(1 row)

 ok_v28 
--------
     26
(1 row)

             ok_v29              
---------------------------------
 2800000000000000000000000000000
(1 row)

